server/src/main/java/org/opensearch/action/admin/indices/template/delete/TransportDeleteIndexTemplateAction.java

```diff
@@ -102,13 +102,13 @@ protected ClusterBlockException checkBlock(DeleteIndexTemplateRequest request, C
     }
 
     @Override
-    protected void masterOperation(
+    protected void clusterManagerOperation(
         final DeleteIndexTemplateRequest request,
         final ClusterState state,
         final ActionListener<AcknowledgedResponse> listener
     ) {
         indexTemplateService.removeTemplates(
-            new MetadataIndexTemplateService.RemoveRequest(request.name()).masterTimeout(request.masterNodeTimeout()),
+            new MetadataIndexTemplateService.RemoveRequest(request.name()).masterTimeout(request.clusterManagerNodeTimeout()),
```

Comment:
Looks like multiple inner classes in `MetadataIndexTemplateService` still need to have the "master" terminology replaced

Reply:
Yes, all the public methods which doesn't belongs to the package or class name with `master` word will be changed in issue #3544